frontend/src/screens/WaitingRoom.js

```diff
@@ -10,8 +10,8 @@ class WaitingRoom extends Component {
   constructor() {
     super();
     this.state = {
-      isUserEnteredName: false,
-      playerList: MOCK_playerList
+      isUserEntered: false,
```

Comment:
player or user?

Reply:
deleted as it get it from context
2fd95b0